tests/Feature/TasksTest.php

```diff
@@ -15,45 +10,21 @@ class TasksTest extends TestCase
     use DatabaseTransactions;
     use RequestHelper;
 
-    const TASKS_URL = '/tasks';
-
-    protected $structure = [
-        'uuid',
-        'updated_at',
-        'created_at',
-    ];
-
-    protected function setUp()
-    {
-        parent::setUp();
-        $this->user = factory(User::class)->create();
-    }
-    
+    const TASKS_URL = '/tasks';    
 
     public function testIndex() {
-        $response = $this->webGet(self::TASKS_URL, []); //is the path correct?
+        $response = $this->webGet(self::TASKS_URL, []); 
         $response->assertStatus(200);
-        $response->assertViewIs('tasks.index'); // is the view correct? this is set within the Controller
-        $response->assertSee('class="container" id="tasks"'); // does the string show within the view?
+        $response->assertViewIs('tasks.index'); 
+        $response->assertSee('class="container" id="tasks"'); 
     }
 
-    // public function testEdit()
-    // {
-    //   $tasks_uuid = factory(User::class)->create()->uuid_text;
-    //   $response = $this->webGet('tasks/'.$tasks_uuid . '/edit');
-    // //   $response->assertStatus(200);
-    //   $response->assertViewIs('tasks.edit');
-    // }
-
-    // public function testCreateRoute()
-    // {
 
-    //   // get the URL
-    //   $response = $this->apiCall('GET', '/admin/users/create');
-
-    //   $response->assertStatus(200);
-    //   // check the correct view is called
-    //   $response->assertViewIs('admin.users.create');
-
-    // }
+    public function testShow() 
+    { 
+        $response = $this->webGet('tasks/id123');
+        $response->assertStatus(200);
+        $response->assertViewIs('tasks.show');
+        $response->assertSee('Approve?');
```

Comment:
This is a dynamic element on the page. Might be better to key off other div id.

Reply:
Updated